app/views/posts/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>New Post</h1>
+
+<%= form_with model: @post do |form| %>
```

Comment:
The user should be able to create a post on the behalf of currently logged-in user. Kindly see the error which occurs while trying to create a new post.